src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figures {

```

Comment:
The class name 'Figures' suggests that it may represent multiple figures, but typically a class represents a single concept or entity. Consider renaming it to 'Figure' if it represents a single figure, or refactoring the design if it is meant to represent a collection of figures.